src/transform_astronomy_data.py

```diff
@@ -0,0 +1,63 @@
+'''Gets JSON data from specified file, filters for relevant columns and converts into dataframe'''
+
+# pylint:disable=line-too-long
+
+import json
+import pandas as pd
+
+FILE_PATH = "data/planetary_data.json"
+
+
+def get_json_data(file_path: str) -> dict:
+    '''Gets JSON from file and returns'''
+    try:
+        with open(file_path, 'r', encoding='utf-8') as file:
+            return json.load(file)
+    except FileNotFoundError:
+        print('Error, No file found')
+        return None
+
+
+def filter_data(json_data: dict) -> pd.DataFrame:
+    '''
+    Takes in JSON and filters for required data as key value pairs in a list of dictionaries
+    Converts dictionary into a dataframe and returns
+    '''
+    longitude = json_data['data']['observer']['location']['longitude']
+    latitude = json_data['data']['observer']['location']['latitude']
```

Comment:
quibble: similar assignments can be made easier to parse by aligning on `=`
```suggestion
    longitude   = json_data['data']['observer']['location']['longitude']
    latitude    = json_data['data']['observer']['location']['latitude']
```